var/spack/repos/builtin/packages/charm/package.py

```diff
@@ -53,13 +53,13 @@ class Charm(Package):
 
     # Communication mechanisms (choose exactly one)
     # TODO: Support Blue Gene/Q PAMI, Cray GNI, Cray shmem, CUDA
-    variant("mpi", default=True,
+    variant("mpi", default=False,
```

Comment:
This isn't a good idea. See #2590 where I had to ensure that every package defaults to `+mpi` since our concretizer is broken.